book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Please make this method to create a new rental, not just adding rentals to an array. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
[Hint] See this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more info.

Reply:
I have kindly implemented those changes 😊

Reply:
Great work implementing the suggested changes. 👏👏👏

However, there are a few more tweaks to make the method perfect.

 - [add_rental should take the person, date and book(self) params and creates a new rental.](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)